alf/utils/tensor_utils.py

```diff
@@ -253,7 +262,7 @@ def clip_by_global_norm(tensors, clip_norm, use_norm=None, in_place=False):
     assert clip_norm.ndim == 0, "clip_norm must be a scalar!"
     assert clip_norm > 0, "clip_norm must be positive!"
 
-    scale = clip_norm / torch.max(clip_norm, use_norm)
+    scale = clip_norm / use_norm
```

Comment:
This will change the behavior. Currently if clip_norm is greater than the vector norm, no scaling will be performed, which means clip_norm is the upper bound of the norm and it doesn't have to be satisfied if the vector norm is already smaller than it.

Reply:
line 275 ensures that nothing happens if clip_norm > use_norm